sync/class.jetpack-sync-module-full-sync.php

```diff
@@ -242,17 +244,27 @@ function update_sent_progress_action( $actions ) {
 		foreach ( Jetpack_Sync_Modules::get_modules() as $module ) {
 			$module_actions     = $module->get_full_sync_actions();
 			$status_option_name = "{$module->name()}_sent";
+			$total_option_name  = "{$status_option_name}_total";
 			$items_sent         = $this->get_status_option( $status_option_name, 0 );
```

Comment:
not a request and probably the safest approach is to keep both, but I wonder if we could get rid of "items_sent"

Reply:
I'm happy to remove it, but also I'm not sure if we use it somewhere else. It would be hard to verify it's no longer used anywhere - maybe it's best to keep it for now, and consider cleaning up in the future.